spec.html

```diff
@@ -38426,56 +38427,56 @@ <h1>Runtime Semantics: UnicodeEscape ( _C_ )</h1>
       </emu-clause>
 
       <emu-clause id="sec-serializejsonobject" aoid="SerializeJSONObject">
-        <h1>Runtime Semantics: SerializeJSONObject ( _value_ )</h1>
-        <p>The abstract operation SerializeJSONObject with argument _value_ serializes an object. It has access to the _stack_, _indent_, _gap_, and _PropertyList_ values of the current invocation of the `stringify` method.</p>
-        <emu-alg>
-          1. If _stack_ contains _value_, throw a *TypeError* exception because the structure is cyclical.
-          1. Append _value_ to _stack_.
-          1. Let _stepback_ be _indent_.
-          1. Set _indent_ to the string-concatenation of _indent_ and _gap_.
-          1. If _PropertyList_ is not *undefined*, then
-            1. Let _K_ be _PropertyList_.
+        <h1>Runtime Semantics: SerializeJSONObject ( _state_, _value_ )</h1>
+        <p>The abstract operation SerializeJSONObject with arguments _state_ and _value_ serializes an object. It performs the following steps:</p>
+        <emu-alg>
+          1. If _state_.[[Stack]] contains _value_, throw a *TypeError* exception because the structure is cyclical.
```

Comment:
and also?
```suggestion
          1. Assert: _state_ is a Record.
          1. If _state_.[[Stack]] contains _value_, throw a *TypeError* exception because the structure is cyclical.
```